flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/flip1/partitionrelease/PipelinedRegionConsumedBlockingPartitions.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.flink.runtime.executiongraph.failover.flip1.partitionrelease;
+
+import org.apache.flink.runtime.jobgraph.IntermediateResultPartitionID;
+
+import java.util.Set;
+
+import static org.apache.flink.util.Preconditions.checkNotNull;
+
+/**
+ * A set of intermediate result partitions that are incident to one {@link PipelinedRegion}.
+ */
+class PipelinedRegionConsumedBlockingPartitions {
+
+	private final PipelinedRegion pipelinedRegion;
+
+	private final Set<IntermediateResultPartitionID> consumedBlockingPartitions;
+
+	PipelinedRegionConsumedBlockingPartitions(
+			final PipelinedRegion pipelinedRegion,
+			final Set<IntermediateResultPartitionID> consumedBlockingPartitions) {
+		this.pipelinedRegion = checkNotNull(pipelinedRegion);
+		this.consumedBlockingPartitions = checkNotNull(consumedBlockingPartitions);
+	}
+
+	public Set<IntermediateResultPartitionID> getConsumedBlockingPartitions() {
```

Comment:
package private method

Reply:
The class already has default visibilty. We could reduce visibility for all it's methods but that makes harder to move the class to a different package if it is needed in the future. Moreover, if a class can be extended,`public` makes it clear what the interface of this class is):

https://stackoverflow.com/questions/5260467/public-methods-in-package-private-classes